lib/models/Bot.js

```diff
@@ -295,6 +295,8 @@ class Bot extends BaseModel {
         const botPocket = await BotItemGeneration.generateBotPockets(parentId, role);
         output.push(botPocket);
         // TODO: generate loot in pockets ? Loose bullets ? Money ? Bandages ? Grenades ?
+        
+        this.generateLootInPockets();
```

Comment:
You aren't parsing pocketContainer in there, will definitely fail.

Reply:
It isn't throwing off any errors by the way